extensions/ql-vscode/src/common/interface-types.ts

```diff
@@ -536,9 +536,8 @@ export interface OpenExtensionPackMessage {
   t: "openExtensionPack";
 }
 
-export interface OpenModelFileMessage {
```

Comment:
This message was unused, so I removed it and added `RefreshExternalApiUsages` in its place.